### PR TITLE
Support the abort option

### PR DESCRIPTION
**Pull request by atheurer**

### CODE REVIEW

File: roadblock.py [hunk 1]
```diff
@@ -239,10 +250,24 @@ def main():
                 else:
                     print("Received ready message from unknown follower '%s'" % (msg[0]))
 
+            if msg[1] == 'abort':
```
Comment (k-rister) [inline roadblock.py]: Should this be
```
elif msg[1] == 'abort':
```

File: roadblock.py [hunk 2]
```diff
@@ -257,6 +282,16 @@ def main():
                 cleanup()
                 print("Exiting")
                 return(0)
+            if msg['data'].decode() == 'go-abort':
```
Comment (k-rister) [inline roadblock.py]: Should this be:

```
elif msg['data'].decode() == 'go-abort':
```

File: roadblock.py (hunk 1 above)
Comment (atheurer) [reply roadblock.py]: fixed

File: roadblock.py (hunk 2 above)
Comment (atheurer) [reply roadblock.py]: fixed

**Pull request merged**